SU-SWS/decanter-web
# nojira | @rebeccahonsf | add questionable vertnav logic and swap out hardcoded nav for component

# READY FOR REVIEW

# Summary
- add questionable but totally works VerticalNav menu tree logic 
- swap out hardcoded nav for component

# Needed By (Date)
- Before ADAPT Dev Chat on Monday so we can converse about the things

# Steps to Test

1. Open preview and confirm that vertical nav displays secondary menu items
2. Review code
